db/mongodb_test.go

```diff
@@ -0,0 +1,76 @@
+package db
+
+import (
+	"os"
+	"testing"
+)
+
+func TestMongoDB(t *testing.T) {
+	id := "33683111000280"
+	json := `{"qsa": [{"name": 42}, {"name": "forty-two"}], "answer": 42}`
+
+	u := os.Getenv("TEST_DATABASE_URL")
+	if u == "" {
+		t.Errorf("expected a posgres uri at TEST_DATABASE_URL, found nothing")
```

Comment:
```suggestion
	u := os.Getenv("TEST_MONGODB_URL")
	if u == "" {
		t.Errorf("expected a mongodb uri at TEST_MONGODB_URL, found nothing")
```

E precisamos atualizar o nome da variável de ambiente no teste do Postgres.